docs/quickstart/rbac.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Role-Based Access Control (RBAC)
+hide_table_of_contents: false
+---
+
+# RBAC
```

Comment:
I'm missing a bit of background context. Why did we build this? Why is it important? How does it help our customers?